shared/zodSchemas.ts

```diff
@@ -369,11 +360,7 @@ export const InstitutionSchema = z.object({
     .min(3, { message: "Institution domain must at least 3 characters " }),
 });
 
-export const createInstitutionSchema = InstitutionSchema.omit({
-  id: true,
-  createdAt: true,
-  updatedAt: true,
-});
+export const createInstitutionSchema = InstitutionSchema.partial();
```

Comment:
Why was this changed

Reply:
Zod was throwing an error even though the data was shaped properly and I didn't wanna spend any more time figuring it out so I threw this in to fix the issue to complete testing